doc/reference/configuration/configuration_reference.rst

```diff
@@ -1447,11 +1447,24 @@ credentials.users.*
 .. confval:: <user_or_role_name>.privileges.lua_call
 
     A list of global user-defined Lua functions that this user or a user with this role can call.
-    To allow calling all such functions, specify the ``all`` value.
+    To allow calling a specific function, specify its name as the value.
+    To allow calling all global Lua functions except built-in ones functions, specify the ``all`` value.
 
     This option should be configured together with the ``execute``
     :ref:`permission <configuration_reference_credentials_privileges_permissions>`.
 
+    To take a previously granted privilege away, specify the ``lua_call`` option without the function name, 
+    or with an empty privileges array (furhter options may retain commented-out).
```

Comment:
I think that a section like 'Revoking priveleges' in https://www.tarantool.io/en/doc/latest/platform/connections_and_auth/credentials/#configuration-credentials would be a more appropriate place for this information.

Reply:
For the record: #5152.